suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -16,6 +16,12 @@ export const networks = {
         features: ['rbf', 'sign-verify', 'amount-unit'],
         customBackends: ['blockbook', 'electrum'],
         accountTypes: {
+            coinjoin: {
+                name: 'Bitcoin Testnet (CoinJoin)',
```

Comment:
It's not a Testnet. And we should use smallcaps for coinjoin, but that could be fixed later separately to keep the "consistency"
```suggestion
                name: 'Bitcoin (coinjoin)',
```